.github/workflows/release.yml

```diff
@@ -288,8 +285,8 @@ jobs:
         DOCKER_TARGET: multi-arch
       run : |
         bin/docker-pull-binaries "$TAG"
-        VERSION=${TAG#"stable-"}
-        mv choco/linkerd.*.nupkg "target/release/linkerd2-cli-stable-$VERSION.nupkg" || true
+        v=${TAG#"stable-"}
+        mv choco/linkerd.*.nupkg "target/release/linkerd2-cli-stable-$v.nupkg" || true
```

Comment:
this seems weird to me... we strip off the stable- prefix and then prepend a stale prefix on the next line... is this to support edges or something as _linkered2-cli-stable-edge-MNP_? Because that seems weird, too.